app/controllers/users/registrations_controller.rb

```diff
@@ -4,13 +4,10 @@ class Users::RegistrationsController < Devise::RegistrationsController
 
   def respond_with(resource, _opts = {})
     if resource.persisted?
-      render json: {
-        status: {code: 200, message: 'Signed up sucessfully.'},
-        data: UserSerializer.new(resource).serializable_hash[:data][:attributes]
-      }
+      render json: { data: UserSerializer.new(resource).serializable_hash[:data][:attributes] }
```

Comment:
I think this is better to be `json: UserSerializer.new(resource).serializable_hash[:data][:attributes]`. So that the frontend can access the data like so: `response.data` instead of `response.data.data`